app/features/utils/functions.js

```diff
@@ -22,3 +23,31 @@ export function openExternalLink(link: string) {
 export function isElectronMac() {
     return process.platform === 'darwin';
 }
+
+/**
+ * Returns the Avatar URL to be used.
+ *
+ * @param {string} key - Unique key to generate Avatar URL.
+ * @returns {string}
+ */
+export function getAvatarURL(key: string) {
```

Comment:
We talked about having this function shared with jitsi-meet, by placing it in js-utils and using it on both places.

Reply:
Yes. I'll modify this when everything else is in place.

Reply:
:+1: